app/scenes/Schedule/index.js

```diff
@@ -15,7 +15,7 @@ import moment from 'moment';
 
 import type { ScheduleTalk } from '../../types';
 
-import Splash from 'react-native-smart-splash-screen';
+// import Splash from 'react-native-smart-splash-screen';
```

Comment:
@brentvatne, thanks for that. I've tried to pull those changes in but I'm still getting an error that I've never seen before. Details are in #44, and the work is on a branch to make it easy to reproduce.